.github/workflows/ci_linux_x64_clang_debug.yml

```diff
@@ -39,23 +39,32 @@ jobs:
       - environment=${{ needs.setup.outputs.runner-env }}
       - cpu
       - os-family=Linux
+    container: gcr.io/iree-oss/base@sha256:dc314b4fe30fc1315742512891357bffed4d1b62ffcb46258b1e0761c737b446
+    defaults:
+      run:
+        shell: bash
     env:
       BUILD_DIR: build-debug
     steps:
       - name: "Checking out repository"
         uses: actions/checkout@v4.1.7
         with:
           submodules: true
-      - name: "Building IREE in Debug configuration"
-        env:
-          IREE_WRITE_REMOTE_CCACHE: ${{ needs.setup.outputs.write-caches }}
+      - name: Install Python requirements
+        run: pip install -r ./runtime/bindings/python/iree/runtime/build_requirements.txt
+      # Note: not using ccache here. Debug builds need a lot of cache space
```

Comment:
NIT:
```suggestion
      # Note: Not using ccache here. Debug builds need a lot of cache space
```

Reply:
🤔 I see 10 results for `# Note: [A-Z]` (uppercase) in the repo and 41 results for `# Note: [a-z]` (lowercase). 

https://developers.google.com/style/capitalization#capitalization-and-colons suggests capitalizing

> Use a lowercase letter to begin the first word of the text immediately following a colon, unless the text is one of the following:
> * ...
> * Text that follows a label such as _Caution_ or _Note_